<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1185</width>
    <height>836</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PhotoChopp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="originalImageView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>550</width>
      <height>550</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="editedImageView">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>550</width>
      <height>550</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="horizontalFilpButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>670</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Horizontal Flip</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>1151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Original Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>560</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Resulting Image</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>610</y>
      <width>1151</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QPushButton" name="revertButton">
     <property name="geometry">
      <rect>
       <x>1050</x>
       <y>120</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(121, 25, 45)</string>
     </property>
     <property name="text">
      <string>Revert
changes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="quantize">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Quantize</string>
     </property>
    </widget>
    <widget class="QPushButton" name="grayscaleButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Grayscale</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tones:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="histogramButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Grayscale
Histogram</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Bright:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="brightnessButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>60</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Brightness</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="brightlevelSpinBox">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-255</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QPushButton" name="contrastButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>60</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Contrast</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Contrast:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="tonesSpinBox">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>256</number>
     </property>
    </widget>
    <widget class="QPushButton" name="histogramEqualizationButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>110</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Histogram
Equalization</string>
     </property>
    </widget>
    <widget class="QPushButton" name="verticalFlipButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Vertical Flip</string>
     </property>
    </widget>
    <widget class="QPushButton" name="histogramMatchingButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>110</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Histogram
Matching</string>
     </property>
    </widget>
    <widget class="QPushButton" name="negativeButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Negative</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rotateClockwiseButton">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>110</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotate Clockwise</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rotateCounterClockwiseButton">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>140</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotate CounterClockwise</string>
     </property>
    </widget>
    <widget class="QPushButton" name="convolutionButton">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>140</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Convolution 3x3</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="kernel00">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>50</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-99.989999999999995</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="kernel02">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>110</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-99.989999999999995</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="kernel12">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>110</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-99.989999999999995</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="kernel22">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>110</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-99.989999999999995</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="kernel21">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>80</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-99.989999999999995</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="kernel11">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>80</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-99.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="kernel01">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>80</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-99.989999999999995</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="kernel10">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>50</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-99.989999999999995</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="kernel20">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>50</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-99.989999999999995</double>
     </property>
    </widget>
    <widget class="QComboBox" name="kernelFilterDropdown">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>20</y>
       <width>211</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Filter Kernel</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gaussian</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Laplacian</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>High-pass</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Prewitt Hx</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Prewitt Hy</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sobel Hx</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sobel Hy</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="zoomInButton">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>60</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ZoomIn</string>
     </property>
    </widget>
    <widget class="QPushButton" name="infoButton">
     <property name="geometry">
      <rect>
       <x>1049</x>
       <y>30</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Info</string>
     </property>
    </widget>
    <widget class="QPushButton" name="zoomOutButton">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>30</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ZoomOut</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="syInput">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>30</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;sy&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sxInput">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>30</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;sx&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>20</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>sx</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>20</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>sy</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="contrastFactorSpinBox">
     <property name="geometry">
      <rect>
       <x>453</x>
       <y>30</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="loadImageButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1151</width>
      <height>541</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>originalImageView</zorder>
   <zorder>editedImageView</zorder>
   <zorder>horizontalFilpButton</zorder>
   <zorder>line</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>loadImageButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1185</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionLoadImage"/>
    <addaction name="actionSaveImage"/>
   </widget>
   <widget class="QMenu" name="menuProcessamento">
    <property name="title">
     <string>Processamento</string>
    </property>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuProcessamento"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoadImage">
   <property name="text">
    <string>Carregar Imagem</string>
   </property>
  </action>
  <action name="actionSaveImage">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Salvar Imagem</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
